__tests__/app.test.js

```diff
@@ -208,4 +208,73 @@ describe('POST /api/articles/:article_id/comments', () => {
             expect(body.msg).toBe('Bad Request')
         })
     });
+});
+describe('PATCH /api/articles/:article_id', () => {
+    test('should return 200 OK status code and respond with votes increased in the updated article', () => {
+        return db.query(`SELECT votes FROM articles WHERE article_id = 6`)
+        .then((votesBeforePatch) => {
+            const numOfVotes = votesBeforePatch.rows[0].votes
+            expect(numOfVotes).toBe(0)
+            return request(app)
+            .patch('/api/articles/6')
+            .expect(200)
+            .send({ inc_votes: 5 })
+        })
+        .then(({ body }) => {
+            expect(body).toEqual({
```

Comment:
Ideally body shouldnt be what the article object exists on, but rather body should have a key of article with the value of the article object itself.